src/main/kotlin/com/katanox/tabour/core/ProtoEventPublisher.kt

```diff
@@ -0,0 +1,12 @@
+package com.katanox.tabour.core
+
+import com.google.protobuf.MessageOrBuilder
+import com.google.protobuf.util.JsonFormat
+import kotlin.reflect.jvm.internal.impl.protobuf.MessageLiteOrBuilder
+
+abstract class ProtoEventPublisher: EventPublisher() {
+    fun publish(event: MessageOrBuilder, busUrl: String) {
```

Comment:
would prefer on relying on a more general solution that takes anything and converts it to JSON